pkg/crd/kubermatic/v1/preset.go

```diff
@@ -346,9 +346,9 @@ func (s EKS) IsValid() bool {
 type AKS struct {
 	ProviderPreset `json:",inline"`
 
-	TenantID       string `json:"tenantId"`
-	SubscriptionID string `json:"subscriptionId"`
-	ClientID       string `json:"clientId"`
+	TenantID       string `json:"tenantID"`
```

Comment:
@maciaszczykm if we change it we have to update presets in the UI. We can also leave like it is right now, WDYT?

Reply:
I'd keep it as it is to be honest. It's the same for Azure I think.

Reply:
OK, so @imharshita please revert those changes for the `preset.go`